### ISSUE TITLE
DOC: Add Bodo to out-of-core projects in ecosystem

### ISSUE THREAD

**Issue by ritwika314** [pandas-dev/pandas#59301]
### Pandas version checks

- [X] I have checked that the issue still exists on the latest versions of the docs on `main` [here](https://pandas.pydata.org/docs/dev/)

### Location of the documentation

https://pandas.pydata.org/community/ecosystem.html

### Documentation problem

 Bodo is a jit compiler that operates on native pandas data frames and supports a large number of APIs https://docs.bodo.ai/latest/api_docs/pandas/, and can be installed as a python package. 

### Suggested fix for documentation

Add bodo to the out of core list of packages in the ecosystem.

**Comment by asishm** [comment]
Thanks for raising this. I have a question that may seem dumb, but I wasn't able to find the answer to. Is the library open-source? Pypi releases seem to have stopped somewhere in 2022, whereas your website documents releases in 2024. I see conda-forge referenced in the quick start, but the conda-forge page seems to go to a private repo.

**Comment by ritwika314** [comment]
Hi @asishm,

Thanks for bringing this up. Bodo is not open-source at present; however, we do offer a free community edition for use with up to 8 cores. We will update the documentation to make it clearer. Do you suggest we add it to the blurb in #59302  as well? 

We will update pip packages in the near future (the team is working through packaging issues in binary dependencies at present)

Regarding conda-forge, [this page](https://anaconda.org/bodo.ai/bodo) has the latest release.

**Comment by mroeschke** [comment]
> Do you suggest we add it to the blurb in https://github.com/pandas-dev/pandas/pull/59302 as well?

Yes, it will be necessary to say there's a free and paid tier of service

**Comment by ritwika314** [comment]
> Yes, it will be necessary to say there's a free and paid tier of service

Done, thank you!
